bin/generate_commit

```diff
@@ -104,6 +104,18 @@ function _create_changelog() {
         --date=iso
 }
 
+function _create_readme(){
+    
+    if [ -e README.md ]; then
```

Comment:
No need to check for the existence of a README, since we're on the `generated-schemas` branch at this point.